### PR TITLE
Add basic Python debugging support

### PR DESCRIPTION
**Pull request by khvn26**

### CODE REVIEW

**Comment by itowlson** [comment]
This is super cool - thank you!

Unfortunately, there are some minor lint failures that are causing CI to fail:

```
ERROR: /home/travis/build/Azure/vscode-kubernetes-tools/src/debug/pythonDebugProvider.ts[44, 17]: Unnecessarily quoted property 'localRoot' found.
ERROR: /home/travis/build/Azure/vscode-kubernetes-tools/src/debug/pythonDebugProvider.ts[45, 17]: Unnecessarily quoted property 'remoteRoot' found.
ERROR: /home/travis/build/Azure/vscode-kubernetes-tools/src/debug/pythonDebugProvider.ts[46, 15]: Missing semicolon
```

If you have time to fix these that would be great; otherwise let me know and I can deal with them.

**Comment by khvn26** [comment]
Done.
As soon as ptvsd team gets pid attach fixed, I'll be happy to extend the provider to be able to attach to any Python pod, regardless of ptvsd runtime availability.
I also plan to think on `isSupportedImage` and `resolvePortsFromFile` heuristics sometime soon, hopefully will figure out something sensible.

**Comment by brendandburns** [comment]
This looks great to me!

**Comment by brendandburns** [comment]
@itowlson are you ok if we merge this?

File: package.json [hunk 1]
```diff
@@ -231,6 +231,18 @@
...
+                            "type": "boolean",
+                            "description": "If true will try to automatically get the root location of the source code in the container (Python)."
+                        },
+                        "vs-kubernetes.python-remote-root": {
+                            "type": "string",
+                            "description": "The root location of the source code in the container (Python)."
+                        },
+                        "vs-kubernetes.python-debug-port": {
+                            "type": "number",
+                            "description": "Remote debugging port for Python. Usually 5678."
```
Comment (itowlson) [inline package.json]: Should we provide defaults for these in the defaults section?

**Comment by itowlson** [comment]
Looks like original author is busy, and the only outstanding comment is very minor, so I'm going to merge this.  Thanks heaps @khvn26!

**Pull request merged**